htdocs/multicurrency/multicurrency_rate.php

```diff
@@ -107,7 +109,7 @@
 
 		$currencyRate_static->fk_multicurrency = $fk_currency;
 		$currencyRate_static->entity = $conf->entity;
-		$currencyRate_static->date_sync = $dateinput;
+		$currencyRate_static->date_sync = $db->idate($dateinput);
```

Comment:
$dateinput is already a timestamp so we should not apply here a database transformation to store the value into memory.